pages/api/v1/dates/registration-closed.ts

```diff
@@ -0,0 +1,39 @@
+import { NextApiHandler, NextApiRequest, NextApiResponse } from 'next';
+import { protect } from '../../../../server/protect';
+import { connectToDatabase } from '../../../../server/mongoDB';
+import { SingletonInfo } from '../../../../common/types';
+
+const regOpenHandler: NextApiHandler = async (req, res) => {
+  switch (req.method) {
+    case 'GET':
+      // stringify bc for some reason, next was removing the quotes in the response
+      await getHandler(req, res);
+    case 'POST':
+      await postHandler(req, res);
+    default:
+      return res.status(405).setHeader('Allow', 'GET, POST').send(undefined);
+  }
+};
+
+const getHandler: NextApiHandler = async (req, res) => {
+  const { dataCollection } = await connectToDatabase('singleton_data');
```

Comment:
we talked about this in the meeting, but after refactoring the DB connection to store two collection clients, you should be able to just grab the other collection, and not have to pass the string to the `connectToDatabase` function.